redisinsight/ui/src/pages/workbench/components/enablement-area/EnablementArea/utils/getFileInfo.ts

```diff
@@ -130,3 +130,29 @@ export const getParentByManifest = (
 
   return parent ?? null
 }
+
+export const findMarkdownPathByPath = (manifest: IEnablementAreaItem[], markdownPath: string) => {
+  const findPath = (data: IEnablementAreaItem[], mdPath: string, path: number[] = []): null | number[] => {
```

Comment:
I suggest change `null | number[]` to `Nullable<number[]>`

Reply:
fixed